doc/developer-documentation/aeproject-cli/fire-editor.md

```diff
@@ -0,0 +1,10 @@
+# aeproject fire-editor
+
+## [Fire Editor aepp](http://fireeditor.nikitafuchs.de/) integration
+
+The integration between the aeproject and the [Fire Editor aepp](http://fireeditor.nikitafuchs.de/) allows the user to compile and deploy contracts, call contract's function from different accounts using the **Fire Editor aepp**. 
+
+The Fire Editor aepp runs on [http://localhost:4200/](http://localhost:4200/) by default. There are two optional parameters to `aeproject fire-editor`:
+
+* --update - update the Fire Editor aepp with the latest version;
+* --ignoreBrowser (--ignorebrowser) - ignoring opening of the browser;
```

Comment:
As far as i remember we have quite a similar parma but still not equal to this one (ignoreOpenInBrowser in the contracts aepp),
Do you think we could keep the params equal for better user experience?